src/pages/Environments.js

```diff
@@ -202,48 +202,43 @@ const Environments = () => {
   const disksByProject = _.groupBy('googleProject', disks)
   const appsByProject = _.groupBy('googleProject', apps)
 
-  const renderBillingProjectApp = app => {
-    const inactive = !_.includes(app.status, ['DELETING', 'ERROR', 'PREDELETING']) &&
-      getCurrentApp(app.appType)(appsByProject[app.googleProject]) !== app
+  const getBillingProjectCell = (billingProject, shouldWarn) => {
     return h(Fragment, [
-      app.googleProject,
-      inactive && h(TooltipTrigger, {
+      billingProject,
+      shouldWarn && h(TooltipTrigger, {
         content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
```

Comment:
All of your workspaces can be linked to the same billing project, so this message doesn't make sense. Did we decide it should read "This Workspace has multiple active…"?

Reply:
Looking at https://github.com/DataBiosphere/terra-ui/pull/2759/files#diff-db07f65dd0ba5edbd535b2a231709c1f5901f4e1c43f39c333165d97c1abacc4R215, I guess the message should technically be "This Google Project has multiple active cloud environments", but the Google Project is a detail that most users are not aware of, and with @gpcar's work we are moving to disallowing multiple disks in the same workspace.

Reply:
This message is also confusing because it's fine to have "multiple active cloud environments" in the same workspace… or Google Project. The problem comes when you have multiple ones of the same TYPE. Like, 2 Galaxy, 2 Cromwell, or 2 GCE in the same Workspace/Google Project. Can we change the message to include that detail like I did in the the multiple persistent disk message?

Reply:
Thanks for pointing this out. Does the following sound correct to you?

> `The workspace has multiple active cloud environments ${forAppText}. Only the most recently created one will be used.`

A `Google project` _can_ have multiple active cloud environments pre-PPW so I don't think we'd want to warn users for that case.

Reply:
Yes, I think that it would move to the Workspace column. And this is one of the places where "cloud environments" gets confusing for apps vs. runtimes. For apps (Galaxy, Cromwell), we are really talking about multiple instances of the app itself… they technically both run in the same kubernetes cluster, which I guess is the cloud environment? For Jupyter/RStudio, we are talking about multiple GCE environments. Coming up with a single wording that works for both might be a challenge.

Reply:
`Cloud Environment` is meant to be an abstraction over backend services like `GKE` and `GCE` to encompass application configuration, cloud compute and persistent disk(s)…so multiple instances of an app running on the same K8s cluster would still be multiple cloud environments.